server/sonar-db-dao/src/test/java/org/sonar/db/DaoModuleTest.java

```diff
@@ -29,6 +29,6 @@
   public void verify_count_of_added_components() {
     ComponentContainer container = new ComponentContainer();
     new DaoModule().configure(container);
-    assertThat(container.size()).isEqualTo(2 + 49);
+    assertThat(container.size()).isEqualTo(2 + 50);
```

Comment:
FYI, magic number `2` can be replaced by constant `ComponentContainer.COMPONENTS_IN_EMPTY_COMPONENT_CONTAINER`